.github/actions/debug-output/action.yaml

```diff
@@ -7,15 +7,18 @@ runs:
     - name: Print basic debug info for a k8s cluster
       run: |
         echo "::group::kubectl get all"
-        uds zarf tools kubectl get all -A
+        uds zarf tools kubectl get all -A | tee /tmp/debug-k-get-all.log
         echo "::endgroup::"
         echo "::group::kubectl get pv,pvc"
-        uds zarf tools kubectl get pv,pvc -A
+        uds zarf tools kubectl get pv,pvc -A | tee /tmp/debug-k-get-pv-pvc.log
         echo "::endgroup::"
         echo "::group::kubectl get package"
-        uds zarf tools kubectl get package -A
+        uds zarf tools kubectl get package -A | tee /tmp/debug-k-get-package.log
         echo "::endgroup::"
         echo "::group::kubectl get events"
-        uds zarf tools kubectl get events -A --sort-by='.lastTimestamp'
+        uds zarf tools kubectl get events -A --sort-by='.lastTimestamp' | tee /tmp/debug-k-get-events.log
```

Comment:
I know we've had issues with the sorting here in the past - not something you introduced - but I'm half tempted to switch this to sort by `metadata.creationTimestamp` or just use `kubectl events -A`. I think creation timestamp would be better for sorting, unsure on `kubectl events`...